CONTRIBUTING.md

```diff
@@ -1,59 +1,91 @@
-
-## Pull Request Process
-
-1. Create a branch. For example, if you are going to work on issue number #44 (which is, say, a new feature for ‘forgot password’ management):
+# Contributing to Circuitverse
+
+- We expect contributors to abide by our underlying [Code of Conduct](https://github.com/CircuitVerse/CircuitVerse/blob/master/code-of-conduct.md) . All discussions about this project must be respectful and harassment-free.
+- Remember that communication is the lifeblood of any Open Source project. We are all working on this together, and we are all benefiting from this software.
+- It's very easy to misunderstand one another in asynchronous, text-based conversations. When in doubt, assume everyone has the best intentions.
+- If you feel anyone has violated our Code of Conduct, you should anonymously contact the team with our abuse report form via [Slack](https://circuitverse.org/slack), necessary action will be take by the team.
+
+## Issue label
+
+Please note if someone is already assigned or working on the issue, do not try to start working without asking in a thread.
+
+- `maintainers` label are internal tasks that will be completed by a Circuitverse core team member. 
+- [good first issue](https://github.com/CircuitVerse/CircuitVerse/labels/good%20first%20issue) labeled issues are meant for newer developers.
+- [feature](https://github.com/CircuitVerse/CircuitVerse/labels/%F0%9F%8C%9F%20feature) labeled issues are meant to propose new features.
+- [bugs](https://github.com/CircuitVerse/CircuitVerse/labels/%F0%9F%90%9E%20bug) labeled issues are meant to have errors in existing code base.
+- [domumentation](https://github.com/CircuitVerse/CircuitVerse/labels/documentation) labeled issues are meant to have typo errors in documentation.
+- [Simulator](https://github.com/CircuitVerse/CircuitVerse/labels/simulator) issues are meant specifically in the scope of circuitverse simulator.
+- [platform](https://github.com/CircuitVerse/CircuitVerse/labels/platform) labeled issues are meant specifically in the scope of circuitverse website.
+- `difficulty: easy` issues are usually confined to isolated areas of existing code.
+- `difficulty: medium` issues sometimes entail new features and might affect a significant area of the codebase, but aren't overly complex.
+- `difficulty: hard` issues are typically far-reaching, and might need architecture decisions during implementation. This label might also denote highly complex issues.
+- `duplicate` labelled issues are meant to be already existing issue in the repository.
+- `priority: less` labelled issues are meant to have priority comparatavily lesser than other issues.
+- `priority: medium` labelled issues are meant to have priority comparatively intermediate than other issues.
+- `priority: high` labelled issues are meant to have highest priority and need to fix as soon as possible.
+- `help wanted` labelled issues are meant to get help from core-team.
```

Comment:
Usually `help wanted` is open to anyone who can help. Maybe we can keep it like that?

Reply:
Should I remove it then?

Reply:
Don't remove it. Just modify it to say
`help wanted` labelled issues signify that the contributor requires help with something specific in the issue and your help is very much appreciated